app/main.py

```diff
@@ -2,36 +2,45 @@
 import pathlib
 import sys
 
-from basyx.aas import model
+from basyx.aas import model, adapter
 from basyx.aas.adapter import aasx
 
 from basyx.aas.backend.local_file import LocalFileObjectStore
 from basyx.aas.adapter.http import WSGIApp
 
-storage_path = os.getenv("STORAGE_PATH", "/storage")
-storage_type = os.getenv("STORAGE_TYPE", "AASX")
+storage_path = os.getenv("STORAGE_PATH", "storage")
+storage_type = os.getenv("STORAGE_TYPE", "LOCAL_FILE_READ_ONLY")
 base_path = os.getenv("API_BASE_PATH")
 
 wsgi_optparams = {}
 
 if base_path is not None:
     wsgi_optparams["base_path"] = base_path
 
-if storage_type == "LOCAL_FILE":
+if storage_type == "LOCAL_FILE_BACKEND":
     application = WSGIApp(LocalFileObjectStore(storage_path), aasx.DictSupplementaryFileContainer(), **wsgi_optparams)
 
-elif storage_type == "AASX":
+elif storage_type in "LOCAL_FILE_READ_ONLY":
     object_store: model.DictObjectStore = model.DictObjectStore()
     file_store: aasx.DictSupplementaryFileContainer = aasx.DictSupplementaryFileContainer()
 
-    for file in pathlib.Path(storage_path).glob("*.aasx"):
+    for file in pathlib.Path(storage_path).iterdir():
         if not file.is_file():
             continue
         print(f"Loading {file}")
-        with aasx.AASXReader(file) as reader:
-            reader.read_into(object_store=object_store, file_store=file_store)
+
+        if file.suffix == ".json":
```

Comment:
Technically, file endings are optional and probably not the best way to check for file type, however I think it's fine to do this here (until we get a bug report).  

Maybe we should do `file.suffix.lower()` though? 
I think I've seen `.JSON` and `.XML` endings on files in the wild.

Reply:
Fixed with https://github.com/rwth-iat/basyx-python-sdk-http-server-docker/pull/2/commits/c743fb260dda0c3a2e805cf909114d769b9136fc